dockers/docker-database/create_single_redis_config.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
>bash [](start = 15, length = 4)

suggest not create many config files for each redis-server instance. it support passing parameter on command line [https://redis.io/topics/config#passing-arguments-via-the-command-line](https://redis.io/topics/config#passing-arguments-via-the-command-line)
Please explore.

Reply:
using command line plus argument instead